docs/self-managed/zeebe-deployment/configuration/broker.md

```diff
@@ -152,6 +154,8 @@ network:
   advertisedHost: 0.0.0.0
   portOffset: 0
   maxMessageSize: 4MB
+  socketReceiveBuffer: 4MB
+  socketSendBuffer: 4MB
```

Comment:
:x: The default value is supposed to be 1MB.

Reply:
Same reasoning as in the first comment.